robyn/router.py

```diff
@@ -53,11 +53,10 @@ def _format_response(
     ) -> Response:
         headers = Headers({"Content-Type": "text/plain"})
 
-        response = {}
         if isinstance(res, dict):
             # this should change
             headers = Headers({})
-            if "Content-Type" not in headers:
```

Comment:
Hey @dave42w , would this not be false always? I think we have a logical bug in Robyn

Reply:
I agree. I was just fixing the